src/main/java/org/apache/commons/lang3/ArrayUtils.java

```diff
@@ -5322,7 +5322,7 @@ static Object removeAll(final Object array, final int... indices) {
      * @return A new array containing the existing elements except the occurrences of the specified element.
      * @since 3.5
      */
-    public static byte[] removeAllOccurences(final byte[] array, final byte element) {
+    public static byte[] removeAllOccurrences(final byte[] array, final byte element) {
```

Comment:
-1 You cannot delete a public API as this will break binary compatibility. You _can_ create a new API with the corrected name and then have the old API delegate to the new one.

Reply:
thanks for your remind! I got it